android/app/src/main/java/app/organicmaps/widget/placepage/sections/PlacePageLinksFragment.java

```diff
@@ -73,10 +81,33 @@ private static void refreshMetadataOrHide(String metadata, View metaLayout, Text
       metaLayout.setVisibility(GONE);
   }
 
-  public static boolean hasLinkAvailable(MapObject mapObject)
+  private static String getLink(@NonNull MapObject mapObject, Metadata.MetadataType type)
```

Comment:
What is the reason to make all these functions static, pass everywhere ```@NonNull MapObject mapObject```, if this mapObject is always a class-member mMapObject? Or not always?

Reply:
Yeah, it looks confusing. I've just tried to make them non-static and found https://github.com/organicmaps/organicmaps/blob/d4d143a9e7f12edd8b82132d584d6327f070a00d/android/app/src/main/java/app/organicmaps/widget/placepage/PlacePageView.java#L317